specification/vmware/Microsoft.AVS.Management/models.tsp

```diff
@@ -595,6 +595,7 @@ model Quota {
 @doc("A location resource")
 @subscriptionResource
 model Location is ProxyResource<LocationProperties> {
+  #suppress "@azure-tools/typespec-azure-core/no-private-usage" "Legacy DO NOT EVER COPY THIS"
```

Comment:
@markcowl @allenjzhang  is that something we can fix here? I would prefer we never have to suppress this one

Reply:
These are likely all cases where we can use existing extensibility points to avoid this.  Will file an issue